src/main/java/se/liu/ida/hefquin/engine/wrappers/lpgwrapper/impl/SingleEdgeLabelMappingToURIsImpl.java

```diff
@@ -0,0 +1,39 @@
+package se.liu.ida.hefquin.engine.wrappers.lpgwrapper.impl;
+
+import org.apache.jena.graph.Node;
+import org.apache.jena.graph.NodeFactory;
+
+public class SingleEdgeLabelMappingToURIsImpl implements EdgeLabelMapping {
+
+    protected final String label;
+    protected final String iri;
```

Comment:
You don't need this member variable anymore. In the `isPossibleResult` function you can get the IRI from the new `node` member variable instead.